src/aluno/PerfilPage.tsx

```
@@ -139,60 +174,133 @@ export function PerfilPage() {
                         </Button>
                     </div>
                     {/* Coluna Direita - Certificados e Projetos */}
-                    <div className="space-y-6">
-                        <div className="space-y-2">
-                            <div className="flex items-center gap-2 text-lg font-semibold text-black">
-                                <FaTools className="w-5 h-5" />
-                                <span>Projetos</span>
+                    <div className="space-y-2">
```

ESLint

[error] Replace `····················` with `··········`

(prettier/prettier)